packages/react/src/NavList/NavList.tsx

```diff
@@ -285,31 +285,23 @@ export type NavListGroupProps = React.HTMLAttributes<HTMLLIElement> & {
   title?: string
 } & SxProp
 
-const defaultSx = {}
-const Group: React.FC<NavListGroupProps> = ({title, children, sx: sxProp = defaultSx, ...props}) => {
-  if (sxProp !== defaultSx) {
-    return (
-      <Box sx={sxProp} as="li" data-component="ActionList.Group">
-        {title ? <ActionList.GroupHeading>{title}</ActionList.GroupHeading> : null}
-        {children}
-      </Box>
-    )
-  }
+function Group({title, children, sx, ...rest}: NavListGroupProps) {
   return (
     <>
       <ActionList.Divider />
-      <ActionList.Group {...props}>
+      <BoxWithFallback as={ActionList.Group} {...rest} sx={sx}>
```

Comment:
I was thinking we probably don't need boxwithfallback here since ActionList.Group would have BoxWithFallback?